adapters/ix/ix.go

```diff
@@ -18,46 +19,69 @@ import (
 	"github.com/prebid/openrtb/v19/openrtb2"
 )
 
+const (
+	FtHandleMultiImpOnSingleRequest = "pbs_handle_multi_imp_on_single_req"
```

Comment:
Nitpicks: There is no need to export this constant (or `ExpireFtTime`) so please change them to starting with a lowercase letter.